src/form/Label/LabelStyle.ts

```diff
@@ -0,0 +1,85 @@
+import styled, { css } from 'styled-components';
+import { srOnlyStyle, themeColor, themeSpacing } from '@amsterdam/asc-ui';
+import { INPUT_SIZE } from '../../shared/constants';
+
+/* @todo Solution wanted: position `bottom` combined with the current position of the input error message seems wrong, possible solution remove position `bottom` */
+
+interface SharedProps {
+	position?: 'top' | 'right' | 'bottom' | 'left';
+}
+
+export type StyleProps = {
+	srOnly?: boolean;
```

Comment:
Bootstrap trucje?

Reply:
Also taken from asc, but perhaps the asc label can be used